apps/nameguard.io/src/app/docs/displayed-name/page.tsx

```diff
@@ -59,12 +59,16 @@ export default function DisplayedNamePage() {
         <div className="divide-y divide-gray-100">
           <div className="grid grid-cols-2 gap-x-6 py-5">
             <div className="w-full flex justify-center items-center font-mono">
-              <pre>{"<TruncatedText name \n maxTooltipWidth={100} />"}</pre>
+              <pre>
+                {
+                  "<TruncatedText name \n textStylingClasses='\nens-webfont font-black' />"
```

Comment:
There's multiple \n here?

Reply:
Yes, is for optimizing text displaying in the screen

Reply:
On who's screen? The developers screen? The user's screen?

This is really strange for me.

Reply:
@lightwalker-eth in both, see how this affects the texts displaying:


https://github.com/namehash/nameguard/assets/49823133/cebe900b-44a0-4dce-a292-c81b69f979a7

The container width is 600px, for screens that are smaller than this the displaying might yet look broken.

For screens wider than this, the displaying is the one we see in the video

Reply:
@FrancoAguzzi Ah, so these newline characters aren't for the actual HTML code, but rather the strings that are in the "Component usage" column?

If so that makes sense to me!

Reply:
Exactly! 👌🏼😃